examples/wheel/wheel_test.py

```diff
@@ -394,11 +395,39 @@ def test_rule_sets_stamped_version_in_wheel_metadata(self):
             version = None
             with zf.open(metadata_file) as fp:
                 for line in fp:
-                    if line.startswith(b'Version:'):
+                    if line.startswith(b"Version:"):
                         version = line.decode().split()[-1]
             self.assertIsNotNone(version)
             self.assertNotIn("{BUILD_TIMESTAMP}", version)
 
+    def test_with_auto_add_requirements(self):
+        if sys.platform.startswith("win"):
+            self.skipTest("This test doesn't work on Windows")
+        filename = os.path.join(
+            os.environ["TEST_SRCDIR"],
+            "rules_python",
+            "examples",
+            "wheel",
+            "my_pytest_wheel-0.0.1-py3-none-any.whl",
+        )
+        with zipfile.ZipFile(filename) as zf:
+            metadata_file = None
+            for f in zf.namelist():
+                if os.path.basename(f) == "METADATA":
+                    metadata_file = f
+            self.assertIsNotNone(metadata_file)
+
+            requires = []
+            with zf.open(metadata_file) as fp:
+                for line in fp:
```

Comment:
Is the order of entries guaranteed? If not, please sort or use a set or comparison.

Reply:
It should be, but I sorted them anyways in e3767dd so that we don't have to care about this in the future in case the code that produces the wheel changes the behaviour.